app/api/v1/mediator/ballot.py

```diff
@@ -172,24 +178,35 @@ def submit_ballots(
     If both are provied, the query string will override.
     """
 
-    manifest, context, election_id = _get_election_parameters(
-        election_id, data, request.app.state.settings
-    )
+    logger.info(f"Submitting ballots for {election_id}")
 
-    # Check each ballot's state and validate
-    ballots = [SubmittedBallot.from_json_object(ballot) for ballot in data.ballots]
-    for ballot in ballots:
+    settings = request.app.state.settings
+    election_sdk = get_election(election_id, settings)
+    manifest_sdk = election_sdk.manifest
+    context_dto = election_sdk.context
+    context_sdk = context_dto.to_sdk_format()
+
+    res: str = ""
+    logger.info(f"Converting {len(data.ballots)} ballots to sdk format")
+    ballots_sdk = list(map(lambda b: b.to_sdk_format(), data.ballots))
+    for ballot in ballots_sdk:
         if ballot.state == BallotBoxState.UNKNOWN:
             raise HTTPException(
                 status_code=status.HTTP_405_METHOD_NOT_ALLOWED,
```

Comment:
I am wondering if 405 is the best choice for an error condition based on the data not being correct.  A 422 Unprocessable Entity might be a better fit overall.